data/core/common.lua

```diff
@@ -277,10 +277,11 @@ end
 
 ---Returns a list of directories that are related to a path.
 ---@param text string The input path.
+---@param root string The root directory.
 ---@return string[]
-function common.dir_path_suggest(text)
+function common.dir_path_suggest(text, root)
   local path, name = text:match("^(.-)([^"..PATHSEP.."]*)$")
-  local files = system.list_dir(path == "" and "." or path) or {}
+  local files = system.list_dir(path == "" and (root or ".") or path) or {}
```

Comment:
`.` might not be useful as we don't `chdir` anymore.
We might need to consider an alternative safe path.

Reply:
I think it might be best to fully remove the fallback, this PR already bumps the mod-version so I doubt this is gonna cause much active harm

Reply:
Okay, so consensus is remove? I can get onboard with that.